### ISSUE TITLE
Problem displaying border in the jet woobuilder plugin cart page.

### ISSUE THREAD

**Issue by 79ho3ein** [CrocoBlock/suggestions#3904]
hello
In the "cart totals"widget  in shopping cart , there is a problem displaying the table border. This margin is displayed on the mobile in some 1px points and in some places 2px. And can not be controlled.  This problem only exists in the size of the mobile phone and is true on the desktop. **To better understand the problem, please see the image below.**
https://prnt.sc/1qnspxh

**theme : hello elementor**

than you

**Comment by 79ho3ein** [comment]
hello @UraraReika 
I explored this problem further, apparently there is a problem displaying the table borders. I do not know exactly what the problem is, but it is there anyway.
In the message above, I reported the problem only on my mobile phone. But when I saw the website on the laptop, there was a problem with the laptop, while on my computer it was not!
This problem is present in the **Cart Table** **Cart Totals** and **Checkout Order Review** widgets.
I hope I have expressed the problem well ...

thank you
